src/go-build-wrapper.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python3
+#
+# Copyright © 2020 – 2021 Red Hat Inc.
+# Copyright © 2022 Ondřej Míchal
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+import os
+import shlex
+import subprocess
+import sys
+
+prog_name = os.path.basename(sys.argv[0])
+
+if len(sys.argv) != 8:
+    print('{}: wrong arguments'.format(prog_name), file=sys.stderr)
+    print('''Usage: {} [SOURCE DIR]
+                           [OUTPUT DIR]
+                           [VERSION]
+                           [C COMPILER]
+                           [DYNAMIC LINKER]
+                           [MIGRATION PATH FORCOREOS/TOOLBOX]
+                           [BUILD FLAGS]'''.format(prog_name), file=sys.stderr)
+    sys.exit(1)
+
+source_dir = sys.argv[1]
+output_dir = sys.argv[2]
+version = sys.argv[3]
+cc = sys.argv[4]
+dyn_linker = sys.argv[5]
```

Comment:
Would be better to use terms like *dynamic_linker*, *dynamic-linker* everywhere.  They are easier to `grep` for (eg., searching for `dynamic` will get you better results), and can defend against [typos](https://github.com/containers/toolbox/pull/942) by making them stand out.